resources/lang/en/admin.php

```diff
@@ -7,5 +7,17 @@
     'accept' => 'Accept',
     'reject' => 'Reject',
     'show' => 'Show',
+    'room' => 'Room',
+    'bed' => 'Bed',
+    'assign_student' => 'Assign a student',
+    'select_floor' => 'Select floor',
+    'select_room' => 'Select room',
+    'first_name' => 'First name',
+    'second_name' => 'Second name',
+    'starting_date' => 'Starting date',
+    'finish_date' => 'Finish date',
+    'assign' => 'Assign',
+    'fourthFloor' => '4. floor',
+    'thirdFloor' => '3. floor',
```

Comment:
I think it will be better to place these in a separate file (not with the admin expressions) since the room allocation should be visible for students also.